<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration_form</class>
 <widget class="QMainWindow" name="registration_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 255, 199);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_title_reg">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-1</y>
      <width>560</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>560</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>560</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>560</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>253</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="default_text">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вспомнили аккаунт -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 255, 220);</string>
       </property>
       <property name="text">
        <string>Вернуться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="creat_ac_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">board: 1px inset blue; 
background-color: rgb(250, 250, 220);</string>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>414</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="default_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="l_text_3">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Логин (Ф.И.О):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new_username">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="default_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="l_text_4">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new_password">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="l_text_5">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Повторный пароль:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replay_password">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
